readme.txt

```diff
@@ -69,6 +69,11 @@ Razorpay is available for Store Owners and Merchants in
 
 == Changelog ==
 
+= 4.6.2 =
+* Fixed, Subscription events webhook bug.
+* Fixed, console errors due to instrumentation.
+* Added Shipping Method Name for Shipping Engine support.
```

Comment:
`Added Shipping Method Name for Shipping Engine support.` It's specific to magic checkout @nkttiwari Can you make it as 
`Added Shipping Method Name for Shipping Engine support in magic checkout`